src/components/DataViewer/SampleDataViewerWithButton.tsx

```diff
@@ -2,7 +2,7 @@
 import React, { useState } from "react";
 import DataViewer, { Data } from "@/components/DataViewer/DataViewer";
 import styled from "styled-components";
-import PhoneIcon from "@/components/Icons/PhoneIcon";
+import PhoneIcon from "@mui/icons-material/Phone";
```

Comment:
please revert this, I would prefer to use already-styled Phone icons rather than MUIs one :)

Reply:
We are not using the icon in our newest iteration